docs/src/documentation/03-components/03-information/badge/03-accessibility.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/badge/accessibility/
+---
+
+## Accessibility
+
+The Badge component has been designed with accessibility in mind.
+
+### Screen reader support
+
+The Badge component supports screen readers through the `ariaLabel` prop. This prop is especially important when:
+
+- The badge contains only an icon
+- The visual content is not sufficient to convey the full meaning
+- Additional context would help understand the badge's purpose
+
+For example:
+
+```jsx
+<Badge ariaLabel="4 passengers" icon={<Icons.Passenger />}>
+  4
+</Badge>
+```
+
+In this example, while sighted users see the number "4" with a passengers icon, screen reader users hear "4 passengers" which provides the complete context.
+
```

Comment:
I would mention, as we always do, to use translated strings for the `ariaLabel`.